docs/conf.py

```diff
@@ -68,7 +65,7 @@ def fetch_parsec_version():
 # the built documents.
 #
 # The short X.Y version.
-version = fetch_parsec_version()
+version = "2.16.0-rc.8+dev"
```

Comment:
This differs from the version in the `.po` files (`Parsec 2.16.0-a.0+dev`). I don't think is important, we can probably change it later in `master`.

Reply:
The CI says no `po` files need to be updated so they will be at a later time ;)